fastapi/app/main_local.py

```
@@ -0,0 +1,287 @@
+        )
+        image_data = thumb_proc.stdout
+        if not image_data:
+            logger.error(f"썸네일 추출 실패: {thumb_proc.stderr.decode(errors='ignore')}")
+            return False
+
+        files = {
+            "file": ("thumbnail.jpg", image_data, "image/jpeg")
+        }
+        # 인증 토큰이 필요하면 여기에 추가
+        headers = {
+            "Authorization": f"Bearer {{your_token_here}}"  # Replace with your token logic
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)